packages/esm-patient-common-lib/src/types/workspace.ts

```diff
@@ -11,5 +11,4 @@ export enum WorkspaceWindowState {
 export interface DefaultWorkspaceProps {
   closeWorkspace(): void;
   patientUuid: string;
-  isTablet: boolean;
```

Comment:
> you also need to remove the place where isTablet gets passed to workspaces. I think this is in workspace.ts.

@brandones is this the one you mentioned?

Reply:
No, I mean here: https://github.com/openmrs/openmrs-esm-patient-chart/blob/4a7fb821c830a0cd055d8b453f9186177586420d/packages/esm-patient-chart-app/src/workspace/workspace-renderer.component.tsx#L40 .

Reply:
It is also already updated

Reply:
You're right! Sorry about that.